shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitParameters.java

```diff
@@ -117,4 +134,65 @@ public ShortCircuitParameters setMinVoltageDropProportionalThreshold(double minV
         this.minVoltageDropProportionalThreshold = minVoltageDropProportionalThreshold;
         return this;
     }
+
+    /** The type of voltage map for the calculation: nominal or previous (voltage values after the loadflow) **/
+    public VoltageMapType getVoltageMapType() {
+        return voltageMapType;
+    }
+
+    public ShortCircuitParameters setVoltageMapType(VoltageMapType voltageMapType) {
+        this.voltageMapType = voltageMapType;
+        return this;
+    }
+
+    /** If the voltage map used for the calculation is nominal, specifies the values taken **/
+    public NominalVoltageMapType getNominalVoltageMapType() {
+        return nominalVoltageMapType;
+    }
+
+    public ShortCircuitParameters setNominalVoltageMapType(NominalVoltageMapType nominalVoltageMapType) {
+        this.nominalVoltageMapType = nominalVoltageMapType;
+        return this;
+    }
+
+    /** Whether the calculation should take into account the resistances into the admittance matrix **/
+    public boolean isUseResistances() {
+        return useResistances;
+    }
+
+    public ShortCircuitParameters setUseResistances(boolean useResistances) {
+        this.useResistances = useResistances;
+        return this;
+    }
+
+    /** Whether the calculation should take into account the loads into the admittance matrix **/
+    public boolean isUseLoads() {
+        return useLoads;
+    }
+
+    public ShortCircuitParameters setUseLoads(boolean useLoads) {
+        this.useLoads = useLoads;
+        return this;
+    }
+
+    /** Whether the calculation should take into account the capacities into the admittance matrix **/
+    public boolean isUseCapacities() {
+        return useCapacities;
+    }
+
+    public ShortCircuitParameters setUseCapacities(boolean useCapacities) {
+        this.useCapacities = useCapacities;
+        return this;
+    }
+
+    /** Whether the calculation should take into account the shunts and model it into the admittance matrix **/
+    public boolean isUseShunts() {
```

Comment:
What do you mean by shunts? All instances of `ShuntCompensator` ? What about static var compensators?